cmd/tempo-cli/cmd-list-cachesummary.go

```diff
@@ -40,12 +40,12 @@ func displayCacheSummary(results []blockStats) {
 	bloomTable := make([][]int, 0)
 
 	for _, r := range results {
-		row := r.CompactionLevel
+		row := r.CompactionLevel //nolint:all //embedded typecheck
```

Comment:
same thought here

Reply:
same as the other one
```
cmd/tempo-cli/cmd-list-block.go:74:46: unifiedMeta.Version undefined (type unifiedBlockMeta has no field or method Version) (typecheck)
	fmt.Println("Version       : ", unifiedMeta.Version)
```
something is wrong with their embedded struct checking

Reply:
yeah, that just seems like a bug. i'd hate to chuck a bunch of nolints in our codebase b/c of a golangci-lint issue. an you track down any info on this issue?

Reply:
OH seems like they fixed it in their last release 8 days ago, will remove this